app/filter.py

```diff
@@ -29,13 +31,51 @@ def __getitem__(self, name):
     def elements(self):
         return self._elements
 
+    def get_theme_file(self):
+        """Returns the contents of the theme file as a string, if it exists. Otherwise returns empty string
+
+        Returns:
+            str: contents of the theme file
+        """
+        contents = ""
+        if self.theme:
+            theme_file = f'app/static/css/themes/{self.theme}.css'
+        if os.path.exists(theme_file):
+            contents = open(theme_file).read()
+        return contents
+
+    def parse_theme_colors(self):
+        """parses the css theme and looks for re-skinning colors. Specifically looks for the .reskin CSS selector.
+        If no selector is found, returns default background 'fff'
+
+        Returns:
+            dict: Key-value dictionary of the properties 
+        """
+        result = {'background-color':'fff'}
+        if self.theme:
+            sheet = cssutils.parseString(self.get_theme_file())
+            if self.dark:
+                selector_name = ".reskin-dark"
+            else:
+                selector_name = ".reskin"
+            reskin_selector = [i for i in sheet if i.selectorText==selector_name]
```

Comment:
Minor styling nitpick here:
```suggestion
            reskin_selector = [i for i in sheet if i.selectorText == selector_name]
```